pkt-line.c

```diff
@@ -254,7 +254,8 @@ int write_packetized_from_fd(int fd_in, int fd_out)
 	return err;
 }
 
-int write_packetized_from_buf(const char *src_in, size_t len, int fd_out)
+int write_packetized_from_buf(const char *src_in, size_t len, int fd_out,
+			      int flush_at_end)
```

Comment:
I have learned that we frequently need to make such an `int <option>` into `int flags` at a later date. Do we want to get ahead of that now, or only do the `flush_at_end -> flags` conversion if/when that happens?

Reply:
I'd rather wait until (or _ifff_) we need it...